WebHostLib/generate.py

```diff
@@ -99,7 +99,12 @@ def start_generation(options: Dict[str, Union[dict, str]], meta: Dict[str, Any])
         except BaseException as e:
             from .autolauncher import handle_generation_failure
             handle_generation_failure(e)
-            return render_template("seedError.html", seed_error=(e.__class__.__name__ + ": " + str(e)))
+            meta["error"] = (e.__class__.__name__ + ": " + str(e))
```

Comment:
all these could be f-strings, ex.
```suggestion
            meta["error"] = f"{e.__class__.__name__}: {e}"
```

Reply:
How'd I miss that one?

Reply:
you can remove the `str` casts in the other copies too 😅